mautic/mautic
# M3: Fix "Recent Activity" feed UI

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/8483
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
On the page overview for a landing page in M3 (on Mautibox), it seems like the line on the page activity history display is a little off-centre and the text is in the wrong place.

![75569109-6bc04b00-5a4c-11ea-8a01-562cf733b88a](https://user-images.githubusercontent.com/43744263/76684575-5c74fc00-660d-11ea-9fa3-3f3d65fad9bd.png)


[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. View any landing page or create a new one.
2. See "Recent Activity" block. Icons and the text are in the wrong place.

#### Steps to test this PR:
1. View any landing page or create a new one.
2. See "Recent Activity" block. Icons and the text must be displayed correctly.
